TestProject/input/Glider.php

```diff
@@ -30,9 +30,9 @@ function fillBoard(Board $_board, GetOpt $_options)
 
         if ($_options->getOption("gliderPosition"))
         {
-            $arguments = explode(',', $_options->getOption("gliderPosition"));
-            $x = $arguments[0];
-            $y = $arguments[1];
+            $gliderPosition = explode(',', $_options->getOption("gliderPosition"));
+            $x = $gliderPosition[0];
+            $y = $gliderPosition[1];
```

Comment:
It just occured to me that php has a nice syntax for these cases. Have a look here:
https://www.php.net/manual/en/function.list.php

This means that this code could be written in a single line like this:
```php
list($x,$y) = explode(',',$_options->getOption("gliderPosition"));
```
Please try it out and explain on the board behind you how this works and what `list(..)` is.